met-web/src/components/engagement/status/index.tsx

```diff
@@ -3,15 +3,15 @@ import { Chip } from '@mui/material';
 import { SubmissionStatus } from 'constants/engagementStatus';
 
 const Open = () => {
-    return <Chip label="Open" color="success" sx={{ fontWeight: 500 }} />;
+    return <Chip label="Open" color="success" sx={{ fontWeight: 700 }} />;
```

Comment:
again back to constant.. change in one place and get the effect everywhere?

Reply:
Fixed it :)